src/components/AssetView/AssetView.stories.tsx

```diff
@@ -0,0 +1,53 @@
+// @ts-ignore
```

Comment:
Why ts-ignore?

Reply:
see other question asking the exact same

Reply:
Ok fine.